Admin/AbstractAdmin.php

```diff
@@ -593,7 +593,7 @@ public function initialize()
     /**
      * {@inheritdoc}
      */
-    public function configure()
+    protected function configure()
```

Comment:
This was on `AdminIntefarce`. I think it should be abstract to keep the same implementation logic.